esp/src/src-react/hooks/grid.tsx

```diff
@@ -174,31 +189,156 @@ export function useFluentGrid({ data, primaryID, alphaNumColumns = {}, query = {
         });
     }, []);
 
-    const renderItemColumn = React.useCallback((item: any, index: number, column: IColumn) => {
-        if (constColumns[column.key].formatter) {
-            return <span style={{ display: "flex" }}>{constColumns[column.key].formatter(item[column.key], item)}</span>;
+    const Grid = React.useCallback(() => <DetailsList
+        compact={true}
+        items={items}
+        columns={fluentColumns}
+        setKey="set"
+        layoutMode={DetailsListLayoutMode.justified}
+        selection={selectionHandler}
+        selectionPreservedOnEmptyClick={true}
+        onItemInvoked={this._onItemInvoked}
+        onColumnHeaderClick={onColumnClick}
+        onRenderDetailsHeader={renderDetailsHeader}
+    />, [fluentColumns, items, onColumnClick, renderDetailsHeader, selectionHandler]);
+
+    const copyButtons = React.useMemo((): ICommandBarItemProps[] => [
+        ...createCopyDownloadSelection(constColumns, selection, `${filename}.csv`)
+    ], [constColumns, filename, selection]);
+
+    return [Grid, selection, copyButtons];
+}
+
+export interface useFluentPagedGridProps {
+    store: ESPRequest.Store,
```

Comment:
I was wondering about this when I was looking through the consumers of this. The useFluentGrid hook takes in an array of rows, where this one takes a store. Would there be any benefit to having them use the same kind of handle to data?